roles/install-k8s/tasks/main.yml

```diff
@@ -96,5 +96,5 @@
   with_items: "{{ groups['all'] }}"
 
 - name: kubeadm join
-  command: "{{ kubernetes_join_command }} {% if (runtime is defined) and 'containerd' == runtime %}--cri-socket /run/containerd/containerd.sock{% endif %}"
```

Comment:
was this is a leftover code from https://github.com/ppc64le-cloud/k8s-ansible/pull/50?

Reply:
This change was not considered for #50. Ideally, it should have been a part of it. :/ 

By docs, it was learnt that kubeadm can auto-detect the runtime and --cri-socket is used in the case of non-standard endpoints or if multiple CRIs are available.